tests/test_models/test_engine/test_db_storage.py

```diff
@@ -0,0 +1,126 @@
+#!/usr/bin/python3
+"""module: dbstorage
+testing methods for dbstorage"""
+
+
+from models.engine import db_storage
```

Comment:
You probably want to be calling `from models import storage` and then set the environment variable `HBNB_TYPE_STORAGE=db` to use `DBStorage` or leave it unset to use `FileStorage`.

Reply:
Is this what you mean?  Right under all the import statements, have:
if getenv("HBNB_TYPE_STORAGE") == 'db':
  DBStorage = db_storage.DBStorage

Reply:
In the `__init__.py` it should be testing the environment variable `HBNB_TYPE_STORAGE` and depending on the value it either sets `storage = DBStorage()` or `storage = FileStorage()` then it calls `storage.reload()` regardless of which one is being used. That way the application is implemented completely the same regardless of the storage backend. There are a few places in the class definitions where you need to test the environment again to get attributes to work as expected but for the most part it is complete storage agnostic.